### PR TITLE
Create Support Page and reference it in FAQ

### PR DESCRIPTION
**Pull request by jodfie**

### CODE REVIEW

**Review by nemchik — CHANGES_REQUESTED** [review]
Please add this to the navigation under basic.
Move the file itself into the basic folder as well.

File: docs/index.md [hunk 1]
```diff
@@ -97,11 +97,7 @@ See our [documentation](https://dockstarter.com/introduction/) for more detailed
 
 ## Support
 
-[![Discord chat](https://img.shields.io/discord/477959324183035936.svg?style=flat-square&color=607D8B&logo=discord)](https://discord.gg/YFyJpmH)
-
-Click the chat badge to join us on Discord for support!
-
-[[Feature Request](https://github.com/GhostWriters/DockSTARTer/issues/new?template=feature_request.md)] [[Bug Report](https://github.com/GhostWriters/DockSTARTer/issues/new?template=bug_report.md)]
+See our [Support Page](basics/support.md)
```
Comment (nemchik) [inline docs/index.md]: This page is also the readme in the main repo. I'd like to keep this info visible. We can still add the link though.

File: mkdocs.yml [hunk 2]
```diff
@@ -11,7 +11,7 @@ repo_url: "https://github.com/GhostWriters/DockSTARTer"
 edit_uri: "https://github.com/GhostWriters/DockSTARTer.com/edit/master/docs/"
 
 # Copyright
-copyright: "Copyright &copy; 2018 - 2019 GhostWriters"
+copyright: "Copyright &copy; 2018 - 2020 GhostWriters"
```
Comment (nemchik) [inline mkdocs.yml]: This is updated already. See my other comment on this file.

File: mkdocs.yml [hunk 3]
```diff
@@ -34,6 +34,7 @@ nav:
       - Available Applications: basics/available-apps.md
       - How to Migrate to DS: basics/migration.md
       - Frequently Asked Questions: basics/faq.md
+      - Support: basics/support.md
```
Comment (nemchik) [inline mkdocs.yml]: Your branch is missing recent changes. The format of the navigation has changed. You'll want to update your local branch by pulling the latest changes. I also recommend using pull bot https://github.com/apps/pull

File: docs/basics/support.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# DockSTARTer Videos, Tutorials, Guides, and Blogs
```
Comment (nemchik) [inline docs/basics/support.md]: With the new changes to the navigation format (mentioned in my other comment) the title of the page (first heading) will be the title used for the navigation link. This is pretty long for that purpose. Can we just call it Support? If so we might need to rename the heading below this.

**Review by nemchik — CHANGES_REQUESTED** [review]

File: docs/basics/support.md (hunk 4 above)
Comment (jodfie) [reply docs/basics/support.md]: support sounds good to me

File: mkdocs.yml (hunk 3 above)
Comment (jodfie) [reply mkdocs.yml]: i'll update... or should it have now that i installed pull bot the other day?

**Pull request merged**